eras/babbage/impl/CHANGELOG.md

```diff
@@ -1,11 +1,13 @@
 # Version history for `cardano-ledger-babbage`
 
-## 1.5.1.1
+## 1.6.1.0
```

Comment:
You can't got from I encourage you to read the changelog part of the [RELEASING.md](https://github.com/input-output-hk/cardano-ledger/blob/master/RELEASING.md#changelogmd) document
```suggestion
## 1.6.0.0
```